code/glTF2Asset.h

```diff
@@ -629,7 +631,7 @@ namespace glTF2
                 float zfar;  //! The floating-point distance to the far clipping plane. (required)
                 float znear; //! The floating-point distance to the near clipping plane. (required)
             } ortographic;
-        };
+        } cameraProperties;
```

Comment:
Is this the right way to name a union?

Reply:
The statement `union { ... } myName;` is a variable declaration, as are `struct { ... } orthographic` and `struct { ... } perspective`. So, in particular, you can't ever create another variable of that `union` type, or of either of the `struct`s you defined in the union, since you haven't given them a type name: they're all three of them anonymous types. If that's what you're looking for, this is perfectly valid syntax.

If, however, you were looking to define a `struct` with the type name `perspective`, you would do something like the following:
```c++
typedef struct perspective {
} perspective;
```
Which, read from the inside out, declares a type `struct perspective`, with whatever members you care to put inside the braces, and defines a type-alias `perspective` to refer to this type, omitting the `struct` portion.

Reply:
I'm… actually not sure what I want in this case lol. I think it was more a fix for a clang warning about anonymous types within anonymous types

Reply:
Hmm...well, do you ever want to be able to do something like `perspective myCamera { ... }; perspective.fov = 90;`? That currently isn't possible. Similarly, it wouldn't be possible to create a `cameraProperties`; the only one that can ever exist is the one you created and named `cameraProperties`.